src/vm/amd64/jithelpers_fast.S

```diff
@@ -110,7 +110,7 @@ LEAF_ENTRY JIT_WriteBarrier, _TEXT
         movabs  rax, 0xF0F0F0F0F0F0F0F0
 
         // Touch the card table entry, if not already dirty.
-        shr     rdi, 0Bh
+        shr     rdi, 0x0B
```

Comment:
For some reason 0FFh is allowed and 0Bh rejected. I'm developing an LLDB plugin for NetBSD to track a process.

For me it looks like a regression in the compiler.. but I don't care about it as 0x is also a valid and used form.